BPMN modeler example is missing dependencies

### Describe the Bug

Building the [modeler example](https://github.com/bpmn-io/bpmn-js-examples/tree/main/modeler) fails because two dependencies (`css-loader` and `style-loader`) are missing in the `package.json`.

### Steps to Reproduce

1. Clone repo
2. cd into `bpmn-js-examples/modeler`
3. Run `npm install`
4. Run `npm run all`

```
npm run all

> bpmn-js-example-modeler@0.0.0 all /root/bpmn-js-examples/modeler
> run-s build


> bpmn-js-example-modeler@0.0.0 build /root/bpmn-js-examples/modeler
> webpack --mode production

assets by status 605 KiB [cached] 2 assets
orphan modules 1.53 MiB [orphan] 378 modules
runtime modules 663 bytes 3 modules
cacheable modules 1.81 MiB
  ./src/app.js + 375 modules 1.53 MiB [built] [code generated]
  ./node_modules/.pnpm/jquery@3.7.1/node_modules/jquery/dist/jquery.js 279 KiB [built] [code generated]

ERROR in ./src/app.js 1:0-44
Module not found: Error: Can't resolve 'style-loader' in '/root/bpmn-js-examples/modeler'

ERROR in ./src/app.js 2:0-41
Module not found: Error: Can't resolve 'style-loader' in '/root/bpmn-js-examples/modeler'

ERROR in ./src/app.js 4:0-61
Module not found: Error: Can't resolve 'style-loader' in '/root/bpmn-js-examples/modeler'

ERROR in ./src/app.js 6:0-21
Module not found: Error: Can't resolve 'style-loader' in '/root/bpmn-js-examples/modeler'

4 errors have detailed information that is not shown.
Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.

webpack 5.90.2 compiled with 4 errors in 5129 ms
 ELIFECYCLE  Command failed with exit code 1.
```

### Expected Behavior

It should build without any modifications to the `package.json`

### Workaround

Run `npm install --save-dev css-loader style-loader` instead of `npm install`.

--- a/commenting/package.json
+++ b/commenting/package.json
@@ -25,6 +25,8 @@
   ],
   "license": "MIT",
   "devDependencies": {
+    "css-loader": "^6.10.0",
+    "style-loader": "^3.3.4",
     "webpack": "^5.89.0",
     "webpack-cli": "^5.1.4",
     "webpack-dev-server": "^5.0.0"

--- a/custom-meta-model/package.json
+++ b/custom-meta-model/package.json
@@ -21,6 +21,8 @@
   ],
   "license": "MIT",
   "devDependencies": {
+    "css-loader": "^6.10.0",
+    "style-loader": "^3.3.4",
     "webpack": "^5.89.0",
     "webpack-cli": "^5.1.4",
     "webpack-dev-server": "^5.0.0"

--- a/deep-linking/package.json
+++ b/deep-linking/package.json
@@ -23,8 +23,10 @@
   ],
   "license": "MIT",
   "devDependencies": {
+    "css-loader": "^6.10.0",
     "npm-run-all2": "^6.0.0",
     "raw-loader": "^4.0.0",
+    "style-loader": "^3.3.4",
     "webpack": "^5.0.0",
     "webpack-cli": "^5.0.0"
   },

--- a/i18n/package.json
+++ b/i18n/package.json
@@ -28,10 +28,12 @@
   "license": "MIT",
   "devDependencies": {
     "copy-webpack-plugin": "^12.0.0",
+    "css-loader": "^6.10.0",
     "eslint": "^8.25.0",
     "eslint-plugin-bpmn-io": "^1.0.0",
     "npm-run-all2": "^6.0.0",
     "raw-loader": "^4.0.0",
+    "style-loader": "^3.3.4",
     "webpack": "^5.0.0",
     "webpack-cli": "^5.0.0",
     "webpack-dev-server": "^5.0.0"

--- a/minimap/package.json
+++ b/minimap/package.json
@@ -28,8 +28,10 @@
   "license": "MIT",
   "devDependencies": {
     "copy-webpack-plugin": "^12.0.0",
+    "css-loader": "^6.10.0",
     "npm-run-all2": "^6.0.0",
     "raw-loader": "^4.0.0",
+    "style-loader": "^3.3.4",
     "webpack": "^5.0.0",
     "webpack-cli": "^5.0.0",
     "webpack-dev-server": "^5.0.0"

--- a/modeler/package.json
+++ b/modeler/package.json
@@ -28,10 +28,12 @@
   "license": "MIT",
   "devDependencies": {
     "copy-webpack-plugin": "^12.0.0",
+    "css-loader": "^6.10.0",
     "eslint": "^8.25.0",
     "eslint-plugin-bpmn-io": "^1.0.0",
     "npm-run-all2": "^6.0.0",
     "raw-loader": "^4.0.0",
+    "style-loader": "^3.3.4",
     "webpack": "^5.0.0",
     "webpack-cli": "^5.0.0",
     "webpack-dev-server": "^5.0.0"

--- a/overlays/package.json
+++ b/overlays/package.json
@@ -26,6 +26,8 @@
     }
   ],
   "devDependencies": {
+    "css-loader": "^6.10.0",
+    "style-loader": "^3.3.4",
     "webpack": "^5.89.0",
     "webpack-cli": "^5.1.4",
     "webpack-dev-server": "^5.0.0"

--- a/transaction-boundaries/package.json
+++ b/transaction-boundaries/package.json
@@ -33,10 +33,10 @@
     "babel-loader": "^9.0.0",
     "babel-preset-env": "^1.7.0",
     "copy-webpack-plugin": "^12.0.0",
-    "css-loader": "^6.8.1",
+    "css-loader": "^6.10.0",
     "npm-run-all2": "^6.0.0",
     "sirv-cli": "^2.0.2",
-    "style-loader": "^3.3.3",
+    "style-loader": "^3.3.4",
     "webpack": "^5.74.0",
     "webpack-cli": "^5.0.0"
   },

